.devcontainer/devcontainer.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "Docker",
   // keep in sync with "run_docker"
-  "image": "pyodide/pyodide-env:20230506-chrome112-firefox112-py311",
+  "image": "pyodide/pyodide-env:20240127-chrome114-firefox122-py311",
```

Comment:
Was there some issue when you used more recent Chrome versions?

Reply:
Oh, it seems like the url for chrome driver has been changed after version 114. I guess we need to update the dockerfile.

![image](https://github.com/pyodide/pyodide/assets/24893111/5c4c3dd4-cb1d-403e-be51-81b9067d7de9)

https://chromedriver.chromium.org/downloads